features/performance/deploy.sh

```diff
@@ -13,30 +13,52 @@ fi
 
 # W/A for https://bugzilla.redhat.com/show_bug.cgi?id=1777150
 # Apply 'worker=""' label only if not set already.
-MCPWORKER=$(oc get mcp --selector=worker="" --no-headers)
-if [ "${MCPWORKER}" == "" ] ; then
-    oc label machineconfigpool/worker worker=
-else
-    echo "MCP/worker already has label 'worker='"
-fi
+until oc label --overwrite machineconfigpool/worker worker=; do
+    sleep 5
```

Comment:
can we find ourselves stuck here in some scenarios ?  or is it ok to keep looping endlessly ? (also applies to the functions below)

Reply:
we can be stuck here only in case when something really bad happened with the environment, but at least from my experience it very rare case and the job has a global timeout.